src/muse/costs.py

```diff
@@ -389,19 +389,38 @@ def annual_levelized_cost_of_energy(
         / techs.utilization_factor
     )
 
-    fuel_costs = (techs.fixed_inputs * prices).sum("commodity")
+    fuel_costs = (
+        convert_timeslice(techs.fixed_inputs, prices.timeslice, QuantityType.EXTENSIVE)
+        * prices
+    ).sum("commodity")
+
+    fuel_costs += (
+        convert_timeslice(
+            techs.flexible_inputs, prices.timeslice, QuantityType.EXTENSIVE
+        )
+        * prices
+    ).sum("commodity")
```

Comment:
Yeah, fuel costs (and any other flexible costs) will depend on production, but it looks like this function is essentially calculating the costs of installing one unit of the technology operating at full capacity. This is why all the exponents are also missing. Whether or not that's the intention I'm not sure...

I also think the treatment of flexible inputs is wrong. Flexible inputs are alternative fuels (i.e. it can use one or the other), so you shouldn't be adding them all together.